automation/taskcluster/decision_task_nightly.py

```diff
@@ -0,0 +1,152 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Decision task for nightly releases.
+"""
+
+from __future__ import print_function
+
+import argparse
+import arrow
+import json
+import lib.tasks
+import os
+import taskcluster
+
+TASK_ID = os.environ.get('TASK_ID')
+SCHEDULER_ID = os.environ.get('SCHEDULER_ID')
+GITHUB_HTTP_REPOSITORY = os.environ.get('MOBILE_HEAD_REPOSITORY')
+HEAD_REV = os.environ.get('MOBILE_HEAD_REV')
+
+BUILDER = lib.tasks.TaskBuilder(
+    task_id=TASK_ID,
+    owner="android-components-team@mozilla.com",
```

Comment:
Nit: do we want a different ML list or can we keep the existing? cc @pocmo

Reply:
Yes, we should use a different owner here (and for notify routes).

@Sdaswani @colintheshots What would be a good mailing list address for this? Ideally that's an address that taskcluster can mail if (Nightly) builds are failing/broken.

Reply:
> @Sdaswani @colintheshots What would be a good mailing list address for this? Ideally that's an address that taskcluster can mail if (Nightly) builds are failing/broken.

@mitchhentges can you use fenix-eng-notifications@mozilla.com ?

Reply:
Yes, that'll work great!